collector-exoplayer/src/main/java/com/bitmovin/analytics/exoplayer/ExoPlayerAdapter.java

```diff
@@ -123,6 +123,25 @@ public void init() {
         this.isVideoAttemptedPlay = false;
         isPlaying = false;
         isPaused = false;
+        checkAutoplayStartup();
+    }
+
+    /*
+      Because of the late initialization of the Adapter we do not get the first couple of events
+      so in case the player starts a video due to autoplay=true we need to transition into startup state manually
+     */
+    private void checkAutoplayStartup() {
+        int playbackState = exoplayer.getPlaybackState();
+        boolean playWhenReady = exoplayer.getPlayWhenReady();
+        if(playbackState == Player.STATE_BUFFERING)
+        {
+            if (playWhenReady != this.isPlaying && playWhenReady) {
+                Log.d(TAG, "Collector was attached while media source was already loading, transitioning to startup state.");
+                // with autoplay enabled the player first enter here and start buffering for the video with playWhenReady = true
+                this.isPlaying = true;
+                startup(getPosition());
```

Comment:
how will this impact the startup measurements? do we want to collect startup-times for these cases?